include/SFML/System/Angle.hpp

```diff
@@ -507,4 +519,9 @@ namespace Literals
 /// sf::Angle a6 = 3.14_rad; // 3.14 radians
 /// \endcode
 ///
+/// Internally, sf::Angle stores the angle value as radians.
+///
+/// For the best performance and to minimize run-time conversions,
+/// it is recommended to work with radians whenever possible.
+///
```

Comment:
I don't necessarily recommend this. I want users to do whatever they find most ergonomic. Besides, it breaks encapsulation to start making recommendations based on implementation details.

Reply:
and also, in many cases there is no cost as the compiler will convert constants at compile time.

Reply:
I think it's good to have this but will leave for a future PR to minimize friction in this one